app/src/main/res/values/strings.xml

```diff
@@ -73,6 +73,7 @@
     <string name="aiMenuRegenerate">Regenerate a response</string>
     <string name="aiMenuSeriousWriting">Serious writing</string>
     <string name="aiMenuShorten">Shorten</string>
+    <string name="aiPromptAnswer">Tell the assistant how he can help you reply to this email.</string>
```

Comment:
```suggestion
    <string name="aiPromptAnswer">Tell the assistant how it can help you reply to this email.</string>
```
Needs to be changed in Loco.